cli/cmd/targetList.go

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2019 Globo.com
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright notice,
+//    this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright notice,
+//    this list of conditions and the following disclaimer in the documentation
+//    and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the copyright holder nor the names of its contributors
+//    may be used to endorse or promote products derived from this software
+//    without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// targetListCmd represents the targetList command
+var targetListCmd = &cobra.Command{
+	Use:   "target-list",
+	Short: "Displays the list of targets, marking the current",
+	Long: `
+Displays the list of targets, marking the current.
+`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		targets := viper.GetStringMap("targets")
+		for k, v := range targets {
+			target := v.(map[string]interface{})
+
+			// format output for activated target
+			if target["current"].(bool) {
+				target["currented"] = "*"
+			} else {
+				target["currented"] = " "
+			}
+
+			fmt.Printf("%s %s (%s)\n", target["currented"], k, target["endpoint"])
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(targetListCmd)
+
+	// Here you will define your flags and configuration settings.
```

Comment:
Are these comments from Cobra? Do you think we need to add them here?

Reply:
Done!